src/commands/drop.ts

```diff
@@ -0,0 +1,137 @@
+import {flags} from '@oclif/command'
+import {BaseCommand} from '../lib'
+import {cli} from 'cli-ux'
+import {getTypesDiff, getFieldsDiff} from '../lib/getdiff'
+
+const DROP_DATA  = `
+mutation {
+  dropData(allData: true) {
+    response { code message }
+  }
+}`
+
+const DROP_SCHEMA  = `
+mutation {
+  dropData(allDataAndSchema: true) {
+    response { code message }
+  }
+}`
+
+const DROP_TYPES = `
+mutation($types: [String!]) {
+  dropData(types: $types) {
+    response { code message }
+  }
+}
+`
+
+const DROP_FIELDS = `
+mutation($fields: [String!]) {
+  dropData(fields: $fields) {
+    response { code message }
+  }
+}
+`
+
+export default class DropData extends BaseCommand {
+  static description = 'Drop all data in your backend'
+
+  static aliases = ['drop-data']
+
+  static examples = [
+    '$ slash-graphql drop-data -e https://frozen-mango.cloud.dgraph.io/graphql -t <apiToken> [-l] [-d] [-s] [-T <types>] [-F <fields>]',
+  ]
+
+  static flags = {
+    ...BaseCommand.commonFlags,
+    ...BaseCommand.endpointFlags,
+    'list-unused': flags.boolean({char: 'l', description: 'List unused types and fields', default: false}),
+    'drop-data': flags.boolean({char: 'd', description: 'Drop data and leave the schema', default: false}),
+    'drop-schema': flags.boolean({char: 's', description: 'Drop Schema along with the data', default: false}),
+    'drop-types': flags.string({char: 'T', description: 'Drop types', multiple: true}),
+    'drop-fields': flags.string({char: 'F', description: 'Drop types', multiple: true}),
+    confirm: flags.boolean({char: 'y', description: 'Skip Confirmation', default: false}),
+  }
+
+  confirm(message: string) {
+    this.log(message)
+    return cli.confirm('Are you sure you want to proceed?')
+  }
+
+  handleResult(errors: [{message: string}] | undefined, successMessage: string) {
+    if (errors) {
+      for (const {message} of errors) {
+        this.error(message)
+      }
+      return
+    }
+
+    this.log(successMessage)
+  }
+
+  async run() {
+    const opts = this.parse(DropData)
+    const backend = await this.backendFromOpts(opts)
```

Comment:
add a check that exactly one of the flags is set (list-unused OR dropschema OR drop-data OR (types AND fields))